debian/jitsi-meet-web-config.templates

```diff
@@ -39,3 +40,13 @@ _Description: Interested in adding telephony to your Jitsi meetings?
  You need to:
   - have a certificate signed by a recognised CA or Let’s Encrypt on your deployment
   - sign in for JaaS at https://jaas.8x8.vc/components?host=${domain}
+
+Template: jitsi-meet/email
+Type: string
+_Description: Enter your email:
+ To successfully issue Let's Encrypt certificates:
+ .
+ You need a working DNS record pointing to this machine(for domain ${domain})"
```

Comment:
How about using hostname everywhere, as we already asked for the hostname at the first installation step?
```suggestion
 You need a working DNS record pointing to this machine(for hostname ${domain})"
```